build/integration-tests.yml

```diff
@@ -1,6 +1,9 @@
 # We create a separate template for these tests because the hosted Windows agents do not support Linux docker containers,
 # and the MS SQL Server team does not provide a docker image whose base image is Windows. Therefore these steps can only run on Linux agents.
 steps:
+- script: docker run -p 10000:10000 -p 10001:10001 -p 10002:10002 -d --rm --name azure-storage mcr.microsoft.com/azure-storage/azurite:latest
```

Comment:
We now run functions integration tests with azurite as the backend. We pass metadata to the tests via environment variables which are loaded as configuration key/values pairs